packages/clerk-js/src/core/resources/User.ts

```diff
@@ -209,6 +209,10 @@ export class User extends BaseResource implements UserResource {
 
   setProfileImage = (params: SetProfileImageParams): Promise<ImageResource> => {
     const { file } = params || {};
+
+    if (file === null) {
+      return Image.delete(`${this.path()}/profile_image`);
+    }
     return Image.create(`${this.path()}/profile_image`, {
```

Comment:
Not part of this PR, but its a little unfortunate that the user knows about the image endpoint 🙃

Reply:
At least it's nested 😌